src/egon/data/datasets/gas_grid.py

```diff
@@ -213,16 +214,108 @@ def insert_CH4_nodes_list(gas_nodes_list):
     )
 
 
-def insert_gas_pipeline_list(gas_nodes_list):
+def insert_gas_buses_abroad(scn_name="eGon2035"):
+    """Insert central gas buses in foreign countries to db, same buses than the foreign AC buses
+    Parameters
+    ----------
+    scn_name : str
+        Name of the scenario
+
+    Returns
+    -------
+    gdf_abroad_buses : dataframe
+        Dataframe containing the gas in the neighbouring countries and one in the center of Germany in test mode
+    """
+    if scn_name == "eGon2035":
+        main_gas_carrier = "CH4"
+    elif scn_name == "eGon100RE":
+        main_gas_carrier = "H2"

```

Comment:
The `main_gas_carrier` could be part of the scenario_parameters table.

Reply:
I changed this, but it still need to be integrated to the CI branch.